src/_store/AppInfoProxy.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import LoadingView from '../loading-view/LoadingView';
+
+@connect(state => ({ appInfo: state.appInfo }))
+export default class AppInfoProxy extends Component {
+    static propTypes = {
```

Comment:
I don't think it matters how many times did we try to reconnect.
But it matters how long was it till we had a connection to the server.

So:
1. Silent retries.
2. After, say 5 seconds of being disconnected, we tell them they are no longer online (5 sec might be too long, we need to test)